### PR TITLE
Feature/add new components

### PR DESCRIPTION
**Pull request by salmma**
Adds prioritization service for NISQ Analyzer feature to quantil-docker such that the NSIQ Analyzer runs without failure.
There is an issue to make the qhana-plugin-runner port editable --> https://github.com/UST-QuAntiL/qhana-plugin-runner/issues/16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +82,7 @@ For running the feature set `winery`, the setup needs additional configuration:
 * `6626`
 * `8080`
 
+
```
Comment (lharzenetter) [inline README.md]: Please remove this unnecessary line.

File: docker-compose.yml [hunk 2]
```diff
@@ -133,6 +135,10 @@ services:
       FOREST_PORT: 5014
       PYTKET_HOSTNAME: pytket-service
       PYTKET_PORT: 5015
+
```
Comment (lharzenetter) [inline docker-compose.yml]: Please remove this unnecessary line.

**Review by lharzenetter — COMMENTED** [review]
Besides additional empty lines, LGTM

File: README.md (hunk 1 above)
Comment (salmma) [reply README.md]: removed

File: docker-compose.yml (hunk 2 above)
Comment (salmma) [reply docker-compose.yml]: removed

**Review by lharzenetter — APPROVED** [review]

**Pull request merged**